core/src/main/java/com/linecorp/armeria/server/logging/Sampler.java

```diff
@@ -83,6 +83,11 @@ public String toString() {
      * @param rate minimum sample rate is 0.01, or 1% of traces
      */
     static Sampler create(float rate) {
+        if (rate == 1.0) {
```

Comment:
I think I miss something but could you tell me the reason why you add this if-else clause even if it is in CountingSampler.create()? :-)